remoulade/helpers/reduce.py

```diff
@@ -17,14 +17,15 @@
 from ..composition import group
 
 
-def reduce(messages, merge_actor, cancel_on_error=False, **kwargs):
+def reduce(messages, merge_actor, cancel_on_error=False, size=2, **kwargs):
```

Comment:
what if your actor has a size parameter ? shouldn't it be something like: 

`def reduce(messages, merge_actor, cancel_on_error=False, size=2, message_kwargs=None):` 

?